development/cli/getting_started.rst

```diff
@@ -72,3 +72,85 @@ Common options that can be used with any of phpBB's CLI commands.
    --no-ansi | Disable ANSI (colors) output
    --no-interaction (-n) | Do not ask any interactive question
    --safe-mode | Run in Safe Mode (without extensions)
+
+Install phpBB using the CLI
+===========================
+
+You need to use the ``install/phpbbcli.php``. Open the file ``docs/install-config.sample.yml`` and copy its content in a new file ``install/install-config.yml``. Change the parameters to your needs. For example a mysql database with ``mysqli`` interface, ``localhost`` and with an database user ``bertie`` with the password ``bertiepasswd`` into the database named ``bertiedb``:
```

Comment:
Can we rewrite this? First we already know about phpbbcli.php in this doc, so...

```rst
The phpBB CLI installer uses a YAML file populated with the data needed to configure a 
database for a new phpBB installation. You can find a sample configuration file in 
``docs/install-config.sample.yml``. Copy this file to ``install/install-config.yml``.
Adjust the sample parameters to your needs.  For example, a MySQL database using 
the ``mysqli`` interface hosted on a ``localhost`` server with the root user ``bertie`` 
and password ``bertiepasswd`` and named ``bertiedb`` would look like:
```